@here/harp-text-canvas/lib/rendering/TextMaterials.ts

```diff
@@ -158,84 +158,116 @@ const sdfTextFragmentSource: string = `
         gl_FragColor = color;
     }`;
 
-function convertShader(shader: THREE.Shader, renderer: THREE.WebGLRenderer) {
-    if (renderer.capabilities.isWebGL2) {
-        shader.vertexShader = convertVertexShaderToWebGL2(shader.vertexShader);
-        shader.fragmentShader = convertFragmentShaderToWebGL2(shader.fragmentShader);
+interface RendererMaterialParameters {
+    rendererCapabilities: THREE.WebGLCapabilities;
+}
+
+interface RawShaderMaterialParameters extends THREE.ShaderMaterialParameters {
+    rendererCapabilities: THREE.WebGLCapabilities;
+}
+
+class RawShaderMaterial extends THREE.RawShaderMaterial {
```

Comment:
Why is this duplicated?

Reply:
When I added RawShaderMaterial, I originally made harp-text-canvas dependent of harp-materials package so that it could use RawShaderMaterial. Nino commented that harp-text-canvas is meant to be an standalone package, that it shouldn't depend on other harp-packages.
So I had to duplicate the class here.